x/wasm/internal/types/genesis.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+// GenesisState is the struct representation of the export genesis
+type GenesisState struct {
+	Codes     []Code     `json:"codes"`
```

Comment:
Code and Contract are only used by GenesisState, correct?
I would move them here, it makes it easier to understand the code